packages/webapp/src/components/ui/TopNav/index.module.scss

```diff
@@ -12,12 +12,7 @@
 	& > a {
 		color: color('text-light') !important;
 		font-weight: 600;
-		position: relative;
-		z-index: 0;
-
-		&:not(:last-child) {
```

Comment:
Yeah, I can cut that, especially with the other styling on username now.

Reply:
Wait, misunderstood the ask here. I took off the special rule because:
1. Doesn't matter with most names vs. where the breakpoints are.
2. Even for very long names, I would like to maintain a margin between the last time in the top nav and the notification bell. This is impart handled by the styling add for the name, so we can avoid the `&:not...` and keep the CSS more consistent/simpler.